Makefile

```
@@ -338,6 +351,7 @@ stop:
 	kill `cat /tmp/redis_cluster_node5.pid` || true
 	kill `cat /tmp/redis_uds.pid` || true
 	kill `cat /tmp/stunnel.pid` || true
+	# kill `cat /tmp/redis_unavailable.pid` || true
```

Need to uncomment

---

NO.
This redis instance is supposed to be shutdown at `UnavailableConnectionTest`.
I've added this line just to trace the instances. I'm not sure whether we should though.

---

But what if the test fails that will leave a rogue running instance, I think we should try and kill it here since this happens *after* running all the tests and would pass still if the process is already dead because of `|| true` bit

---

AFAICR, I commented it because it failed.

---

You're right, Redis deletes the pid file on exit, how about this to check if the file exists first:
```suggestion
	[ -f /tmp/redis_unavailable.pid ] && kill `cat /tmp/redis_unavailable.pid` || true
```